x-pack/plugins/search_assistant/server/functions/index.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RegistrationCallback } from '@kbn/observability-ai-assistant-plugin/server';
+
+export const registerFunctions: RegistrationCallback = async ({
+  client,
+  functions,
+  resources,
+  signal,
+}) => {
+  const isServerless = !!resources.plugins.serverless;
+
+  functions.registerInstruction({
+    instruction: `You are a helpful assistant for Elasticsearch. Your goal is to help Elasticsearch users accomplish tasks using Kibana and Elasticsearch. You can help them construct queries, index data, search data, use Elasticsearch APIs, generate sample data, visualise and analyze data.
```

Comment:
Q: Did we consider translations here? Or is the assistant an English-only feature for now?

Reply:
English-only for now.